Corona-Warn-App/src/main/res/navigation/covid_certificates_graph.xml

```diff
@@ -310,6 +310,10 @@
             android:id="@+id/action_dccReissuanceConsentFragment_to_dccReissuanceSuccessFragment"
             app:destination="@id/dccReissuanceSuccessFragment" />
 
+        <argument
+            android:name="personCode"
```

Comment:
Rename to `personIdentifiercode` to be consistent?

Reply:
Could we actually pass the `CertificateReissuance` object from the `DccWalletInfo` instead so we don't need to get it in the Consent Screen again?